types/student/student-entity.ts

```diff
@@ -29,6 +29,15 @@ export interface StudentImport {
   bonusProjectUrls: string;
 }
 
+export interface SingleStudentImport {
+  email: string;
+  courseCompletion: number;
+  courseEngagement: number;
+  projectDegree: number;
+  teamProjectDegree: number;
+  bonusProjectUrls: string;
+}
+
```

Comment:
I think that the student id could also be very useful on FE, when redirecting from the list of students to the profile of a specific student using the id in the URL.